index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute to Dr. Aklilu Lemma</title>
+  </head>
+  <body>
+    <main id="main">
+      <div id="heading" class="center">
+        <h1 id="title" class="center">Dr. Aklilu Lemma</h1>
+        <p class="font"><i>The man who was an Ethiopian pathobiologist</i></p>
+      </div>
+      <div id="img-div">
+        <img
+          class="center"
+          id="image"
+          src="Aklilu Lema.jpg"
+          alt="Aklilu Lemma with his Invention"
+        />
+        <div class="center">
+          <br />
+          <p id="img-caption" class="font">
+            Aklilu Lemma in his invention of Drugs...
+          </p>
+          <br />
+        </div>
+      </div>
+      <div class="div1">
+        <div id="tribute-info" class="center">
+          <h2 class="font">Here's a time-line of Aklilu Lemma's life:</h2>
+          <ul class="font" style="text-align: justify">
+            <br />
+            <li>
+              <b>1935</b> - born september 1935, in Jigjiga, Hararghe Province.
+            </li>
+            <br />
+            <li>
+              <b>1964</b> - After graduating from Addis Ababa university, He
+              made his most important scientific discovery very early in his
+              career in 196.
+            </li>
+            <br />
+            <li>
+              <b>1989</b> - The most prestigious award was the one he won
+              together with his research associate, Dr. Legesse Wolde-Yohannes
+              in November 1989 -the Right Livelihood Award of Sweden, commonly
+              referred to as the Alternative Nobel Prize.
+            </li>
+            <br />
+            <li>
+              <b>1997</b> - He died in the United States on 5 April 1997 and was
+              buried in Ethiopia, on 13 April.
+            </li>
+          </ul>
+        </div>
+
+        <div class="center">
```

Comment:
Please remove blank spaces

Reply:
- [ ] I think here you can say it also in a polite tone and mentioning the reason why you asked that like instead of saying 

> Please remove blank spaces


you can say:


> Why you don't remove the blank spaces, I think it would be better to remove them to make your code clean.🔥